gp-includes/assets-loader.php

```diff
@@ -1,84 +1,94 @@
 <?php
 /**
- * Defines default styles and scripts
+ * Defines default styles and scripts.
+ *
+ * @package GlotPress
+ * @since 1.0.0
  */
 
 /**
- * Register the GlotPress styles
- *
- * @param WP_Styles $styles
+ * Register the GlotPress styles and load the base style sheet.
```

Comment:
Docs are third-person singular elements, so third-person singular verbs should be used. (https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#language)